pages/domains/domains_eligibility.md

```diff
@@ -27,12 +27,12 @@ If you’re eligible to have a .gov domain, we want you to get one. The types of
 After you request a .gov domain, we'll review the information you provided about your organization. We use the [U.S. Census Bureau’s criteria for classifying governments](https://www.census.gov/programs-surveys/gus/technical-documentation/methodology/population-of-interest1.html){.usa-link--external} to help determine eligibility. In some cases, we’ll ask for more information (such as legislation, a charter, or bylaws) to verify eligibility.
 
 
-## You must have approval from an authorizing official within your organization
-Your authorizing official is a person within your organization who can authorize your domain request. This person must be in a role of significant, executive responsibility within the organization.
+## You must have approval from an senior official within your organization
```

Comment:
This should be "**a** senior official" - not updating because I think that will break links in various places. Can you update here and in the links?

Reply:
Here are the new links to click through!

[Before you request](https://federalist-877ab29f-16f6-4f12-961c-96cf064cf070.sites.pages.cloud.gov/preview/cisagov/get.gov/rh/2259-update-ao-to-so/domains/before/) (this one has two links on this page for the senior official portion which should lead to the link below!)
[The main link for approval from a senior official](https://federalist-877ab29f-16f6-4f12-961c-96cf064cf070.sites.pages.cloud.gov/preview/cisagov/get.gov/rh/2259-update-ao-to-so/domains/eligibility/#you-must-have-approval-from-a-senior-official-within-your-organization)